crates/ibc/Cargo.toml

```diff
@@ -73,6 +73,8 @@ parity-scale-codec = { version = "3.0.0", default-features = false, features = [
 scale-info = { version = "2.1.2", default-features = false, features = ["derive"], optional = true }
 ## for borsh encode or decode
 borsh = {version = "0.9.3", default-features = false, optional = true }
+parking_lot = { version = "0.12.1", default-features = false }
+cfg-if = "1.0.0"
```

Comment:
Could you move those under `[dev-dependencies]`?

Reply:
I don't think it should be transferred over, because opening the mocks feature brings in some needed code

Reply:
Ah yes you're right

Reply:
Then can you make them optional, and a dependency of `mocks`?